src/server/server.h

```diff
@@ -53,6 +53,8 @@
 #include "tls_util.h"
 #include "worker.h"
 
+constexpr const char *REDIS_VERSION = "4.0.0";
```

Comment:
why so far back? is kvrocks compatible with up to redis 4 ? not more than that?

Reply:
@asad-awadia It's the least promised compatibility version, we also would like to retrospect if need to update this version since most 6.0/7.0 commands are supported now. But it'd be better to discuss this in another place.